Add in tarball upload to UNL repository

timtheisen opened the pull request.

timtheisen commented:
I have cut and pasted the script and ensure that it works properly.

brianhlin commented on docs/release/cut-sw-release.md:
```diff
@@ -310,17 +310,40 @@ NON_UPCOMING_VERSIONS="<NON-UPCOMING VERSION(S)>"
 
+Ask Tim Theisen, Brian Lin, or someone with privileges on the `opensciencegrid.org` repo servers to upload the tarballs with the following procedure:
+
+#### On a CS machine
```
Does everyone on S&R have SSH access to `hcc-osg-software.unl.edu`? If they do not, what's the process for getting access?

brianhlin commented on docs/release/cut-sw-release.md:
```diff
@@ -310,17 +310,40 @@ NON_UPCOMING_VERSIONS="<NON-UPCOMING VERSION(S)>"
+    major_ver="${ver%.*}"
+    ssh osgcollab@hcc-osg-software.unl.edu "cd /usr/local/repo/tarball-install; ls -l $major_ver/*latest*"
+done
+# verify the symlinks are correct
```
This comment and the one above can be clearer. What's correct?

brianhlin commented on docs/release/cut-sw-release.md:
```diff
@@ -310,17 +310,44 @@ NON_UPCOMING_VERSIONS="<NON-UPCOMING VERSION(S)>"
+
+!!! note
+    If you do not have access to the tarball repository at UNL (osgcollab@hcc-osg-software.unl.edu),
+    please forward your public SSH key to Derek Weitzel.
```
This may not be decided yet but what if Derek is OOO? Is there a list that we can use instead?

Also, let's move this note to the top of the section so users aren't doing a partial client tarball update and copy its text to the `Requirements` section up top.

timtheisen commented:
I moved the UNL repo requirement to the requirement section. I dropped the SSH public key statement altogether. Any with access to the repo and grant another team member access. Also, we don't get into how to satisfy these requirement in this document.
